server/fleet/service.go

```diff
@@ -753,11 +753,14 @@ type Service interface {
 	RequestEncryptionKeyRotation(ctx context.Context, hostID uint) error
 
 	///////////////////////////////////////////////////////////////////////////////
-	// Microsoft MDM
+	// Windows MDM
 
 	// GetMDMMicrosoftDiscoveryResponse returns a valid DiscoveryResponse message
 	GetMDMMicrosoftDiscoveryResponse(ctx context.Context) (*DiscoverResponse, error)
 
+	// GetMDMMicrosoftPolicyResponse returns a valid GetPoliciesResponse message
+	GetMDMMicrosoftPolicyResponse(ctx context.Context, authToken string) (*GetPoliciesResponse, error)
```

Comment:
Same here? (microsoft vs windows)

Reply:
good catch! I missed this one. Just updated the name